tests/composer.json

```diff
@@ -8,12 +8,12 @@
     "require-dev": {
         "squizlabs/php_codesniffer": "1.*",
         "phpmd/phpmd": "*",
-        "codeception/codeception": "~2.1",
+        "codeception/codeception": "~2.2@dev",
         "phing/phing": "2.9.*@dev",
         "joomla-projects/joomla-browser": "v3.4.8.2",
         "codegyre/robo": "~0.6",
         "redcomponent/robo": "dev-master",
-        "redcomponent/teststatistics": "dev-master",
+        "redcomponent/teststatistics": "2.2",
```

Comment:
as far as I know, this is outdated and doesn't work with the latest version of Codeception, please check it again @PhamHanh96 @nhungredweb

Reply:
Thanks @puneet0191 I will check and fix it now. Thanks so much